common/task.go

```diff
@@ -9,6 +9,11 @@ type TaskContainerLister interface {
 	ListTasks(environment string, serviceName string) ([]Task, error)
 }
 
+// TaskRestarter for restarting tasks
+type TaskRestarter interface {
```

Comment:
Although it is correct to name the CLI and workflow as "restarter", seems this interface is just to stop tasks.  Would prefer the name be `TaskStopper`